pages/banks/[bankTag].vue

```diff
@@ -1,19 +1,7 @@
 <template>
-    <!-- :summary, :header, :details, :amountFinancedSince2016 to be removed 
-        https://linear.app/bankgreen/issue/PE-476/render-custom-bank-texts-from-prismic
-    -->
-    <component v-if="details" :is="componentName"
-        :name="details.name" 
-        :subtitle="details.subtitle"
-        :website="details.website" 
-        :inheritBrandRating="details.inheritBrandRating" 
-        :header="details.header"
-        :summary="details.summary" 
-        :details="details.details" 
-        :amountFinancedSince2016="details.amountFinancedSince2016"
-        :fossilFreeAlliance="details.fossilFreeAlliance"
-        :bankPage="bankPage"
-    />
+    <component v-if="details" :is="componentName" :name="details.name" :website="details.website"
```

Comment:
Weird indentation here. Is there a standard for that?

Reply:
It's Prettier, which enforces a max line-length I reckon